app/src/main/java/org/mozilla/tv/firefox/webrender/WebRenderFragment.kt

```diff
@@ -256,8 +257,6 @@ class WebRenderFragment : EngineViewLifecycleFragment(), Session.Observer {
             }
 
             registerForContextMenu(browserOverlay.tileContainer)
```

Comment:
Maybe some of these later lines that rely on the view having been created should be moved into `onViewCreated`, so we don't have any more assumptions on whether the views are there or not.

Reply:
To be fair the views are available at this point, the problem behind this crash was the VM not being available.  It's probably a good policy to set up VMs before modifying view state, but I think we're safe keeping them in the same method.  I don't feel strongly about it though, I can move them if you'd prefer.

Reply:
Hmm, I guess that's a good point but the VMs do back our views - I guess technically `onViewCreated` is more of an Android lifecycle thing. But, still a slight preference to move them to `onViewCreated` so we can clearly separate modifying view state from setting it up.